scss/_footer.scss

```diff
@@ -0,0 +1,72 @@
+body .container {
+  padding: 1em 1em 0em 1em !important ;
+}
+
+#s-lc-public-footer {
+  background-color: $text_black;
+  border-top: none;
+  margin: 0 auto;
+  font-size: 100%;
+  color: $text_footer_gray;
+  padding: 45px 0 0 0;
+  line-height: 1.75;
+}
+
+#s-lc-public-footer-admin-links,
+#s-lc-public-footer-brand,
+#s-lc-footer-support-link,
+#s-lc-language,
+.public-footer-container {
+  a {
+      color: $text_footer_dark_gray;
+      text-decoration: underline;
+      &:hover {
+          color: $text_footer_white;
+      }
+  }
+  select {
+      color: $text_black;
+  }
+  select option {
+      color: $text_black;
+  }
+}
+
+#s-lc-language {
+  padding: 7px 0px 10px 0px;
+}
+
+#s-lc-public-cust-footer {
+  width: max-content;
+  margin: auto;
+}
+
+@media only screen and (max-width: 830px){
```

Comment:
@laurxnemeth could you move these breakpoint values (i.e. 830px, etc.) into `_variables.scss` and resolve with the existing breakpoints marked as `$bp_`